coretex/bioinformatics/__init__.py

```diff
@@ -52,19 +53,18 @@ def cutadaptTrim(
 
     args: list[str] = [
         "cutadapt",
-        "-o", forwardOutput,
+        "-o", str(forwardOutput),
```

Comment:
do it like this for all of them `if isinstance(forwardOutput, Path): forwardOutput = str(forwardOutput`. We don't wanna convert it to `str` if it is already `str` object